### DISCUSSION TITLE
feature request: skip_full_temporary

### DISCUSSION THREAD

**Discussion by Elbraddampfer** [swar/Swar-Chia-Plot-Manager#494] [Ideas]
Hi,
I would like it if there were also a _**skip_full_temporary**_ as a parameter. The background is that I have already been in this situation twice and the manager started new plots although it was foreseeable that the space on the temp-drive was not enough. This happens when you set a stagger time that is too short and old jobs have not yet been cleared while new jobs are being started. With such a parameter, this freezing could be avoided.

**Comment by swar** [comment]
I don't think I will be tackling this functionality. There's a lot of features in play to prevent this kind of stuff from happening already. 

Not only that, but what constitutes a full temporary directory? If it's full, it will stop all work already. I think the proper answer is to tweak the config file to prevent this scenario.

**Reply by Elbraddampfer** [reply]
By full I did not mean the drive actually filled to 100% but the manager's check whether the desired plotfile (e.g. k32) even fits on the temp directory when it is started. If there is not enough space it could just wait for the stagger time and then check it again and start it when the temp is cleaned up.
